docs/developer-guide/connect.rst

```diff
@@ -0,0 +1,144 @@
+.. _ksql-connect:
+
+KSQL-Connect Integration
+========================
+
+`Kafka Connect`_ is an open source component of Kafka that simplifies loading and exporting data
```

Comment:
For links to internal docs, we like to use (logical) anchors rather than (physical) URLs, so in this case the link target is: 
```
:ref:`kafka_connect`
```